src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,33 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+//ciudad actual es de tipo ciudad
+class CentroDeDistribucion(val ciudadActual: Ciudad) {
+    //vendedoresConLosQueTrabajan es de tipo lista
+    val vendedoresConLosQueTrabajan = mutableListOf<Vendedor>()
+
+    //no devuelve nada
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (this.estaRegistrado(vendedor)){
+            throw Exception("el vendedor está registrado")
+        }
+        vendedoresConLosQueTrabajan.add(vendedor)
+    }
+    //No devuelve nada
+    fun sacarVendedor(vendedor: Vendedor) = vendedoresConLosQueTrabajan.remove(vendedor)
+
+    //devuelve boleano
+    fun estaRegistrado(vendedor: Vendedor) = vendedoresConLosQueTrabajan.contains(vendedor)
+
+    //devuelve el vendedor con mayor puntaje
+    fun vendedorEstrella() = vendedoresConLosQueTrabajan.maxBy { vendedores -> vendedores.puntajeCertificaciones()}
```

Comment:
Al ser una lista de vendedores cada elemento de la lista va a ser un `Vendedor`, por lo que sería más correcto algo así
```suggestion
    fun vendedorEstrella() = vendedoresConLosQueTrabajan.maxBy { vendedor -> vendedor.puntajeCertificaciones()}
    //O también podés hacer esto y te ahorras espicificar un nombre
    fun vendedorEstrella() = vendedoresConLosQueTrabajan.maxBy { it.puntajeCertificaciones()}
```